osu.Game/Overlays/Notifications/ProgressNotification.cs

```diff
@@ -252,6 +252,11 @@ private void load(OsuColour colours, AudioManager audioManager)
             cancelSample = audioManager.Samples.Get(@"UI/notification-cancel");
         }
 
+        /// <summary>
+        /// Dismisses the notification immediately without any animation.
```

Comment:
This doesn't work well for me. It's not just closing immediately. It's closing and skipping cancelling the task, going against the normal close flow.

I'd rename this to `DismissWithoutCancelling` and update the xmldoc accordingly.